packages/foreman/tfm/tfm.spec

```diff
@@ -131,6 +131,12 @@ Obsoletes: tfm-ror52-rubygem-turbolinks-doc < 2.5.4-3
 Obsoletes: tfm-ror52-rubygem-arel <= 9.0.0
 Obsoletes: %{scl_prefix}-rubygem-arel <= 9.0.0
 
+# obsolete all of the tfm Passenger packages since Passenger support is dropped
```

Comment:
I think it should also do the same with doc packages but not entirely sure.

Reply:
Done.